README.md

```diff
@@ -40,10 +64,78 @@ Fetch credentials from CyberArk Conjur using the controlling host's Conjur ident
       msg: "{{ lookup('cyberark.conjur.conjur_variable', '/path/to/secret') }}"
 ```
 
-### Author Information
-- CyberArk Business Development Technical Team 
-    - @cyberark-bizdev
-    - @enunez-cyberark
-    - @jimmyjamcabd
-- CyberArk Community and Integrations Team 
-    - @cyberark/community-and-integrations-team
+## Conjur Ansible Role
+This Ansible role provides the ability to grant Conjur machine identity to a host. Based on that identity, secrets can then be retrieved securely using the [Summon](https://github.com/cyberark/summon) tool (installed on hosts with identities created by this role).
+
+### Usage
+The Conjur role provides a method to "Conjurize" or establish the Conjur identity of a remote node with Ansible. The node can then be granted least-privilege access to retrieve the secrets it needs in a secure manner.
+
+### Role Variables
+
+* `conjur_appliance_url` `*`: URL of the running Conjur service
+* `conjur_account` `*`: Conjur account name
+* `conjur_host_factory_token` `*`: [Host Factory](https://developer.conjur.net/reference/services/host_factory/) token for
+layer enrollment. This should be specified in the environment on the Ansible controlling host.
+* `conjur_host_name` `*`: Name of the host being conjurized.
+* `conjur_ssl_certificate`: Public SSL certificate of the Conjur endpoint
+* `conjur_validate_certs`: Boolean value to indicate if the Conjur endpoint should validate certificates
+* `summon.version`: version of Summon to install. Default is `0.6.6`.
+* `summon_conjur.version`: version of Summon-Conjur provider to install. Default is `0.5.0`.
```

Comment:
are these optional parameters?

Reply:
I see the comment below - I actually find it clearer when the definition includes `(optional)` vs having the asterisks, up to you which you go with

Reply:
also, are these defaults for real? they're so outdated!

Reply:
Totally - I can bump those versions